### DISCUSSION TITLE
Flashing the firmware seems not to work with sv06+

### DISCUSSION THREAD

**Discussion by minaf1** [bassamanator/Sovol-SV06-firmware#116] [General]
Hello @bassamanator.

I am new to this 3d printing craze, and trying to install klipper on my sv06+ printer.
I tried to follow your readme on the sv06+ branch. I believe i've gotten the same error you mentioned saying, "Many users have reported having issues flashing Klipper using the Sovol microSD card". 
What I've done is reformat the sd card and copy/paste your bin file (by itself) in the card.  I tried this using sovol's sd card as well as my own separate 32gb sd card. In both cases the printer ends up freezing as in the screen doesn't start up (I can share a picture if it would help).  Is there a solution known for this? or sd cards that should work with this printer (preferably with klipper)?

**Comment by bassamanator** [comment]
Unfortunately, according to Sovol3d official, you cannot use a card that is larger than 16GB (see the video in that section). So your card should not work because it's 32GB, and their card doesn't work for many users and you might be one of these users.

I don't think you've read all the documentation as is required to make this whole thing happen. I say that because it's actually not possible to tell at that point in the readme if your flash was successful.

**Reply by minaf1** [reply]
No, its not possible to tell whether it was successful, since the flash did not complete. It stayed frozen on the sv06+'s screen without starting up, even after waiting 20 minutes. Do you have any recommended brand and size microSD card that will work with the sv06+?

**Comment by bassamanator** [comment]
Did you proceed onto the download section and complete those steps?

Here's what I currently use: https://www.amazon.ca/gp/product/B0858FBL8V/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1. Any small size microsd card _should_ work.

Note that you should be able to re-flash marlin using Sovol's microsd card.

**Reply by minaf1** [reply]
yes, I did clone the repo and copy the bin file within the misc directory, if this is what you're referring. this is the file I copied: klipper-v0.12.0-10-gea2f6bc0.bin.  

right, but I want to get klipper working. So far, this printer has been not much more than a paper weight for me, failing the simplest builds like benchy. I have a suspicion that it could have to do with the table the printer is sitting on. The table is on wheels so each time the printer moves, it vibrates the entire table, possibly throwing off the print.  I may have to try a test print without the table =).

Clearly, klipper is not the solution that would fix this problem. If anything, I imagine getting klipper running would exacerbate the table issue worse, since small motions don't make much difference, but the jerking motions absolutely would (or are more likely to).

**Comment by bassamanator** [comment]
The table is fine. Look up "Maker's muse" channel on Youtube where takes talks about his shaky table. He's an expert btw.

Klipper is not a solution for whatever problems you're experiencing, it's a firmware.

If you're actually trying to get going on printing, the first step is to run through the initial steps mentioned on my repo here: https://github.com/bassamanator/everything-sovol-sv06.

It's apparent to me that you didn't read the documentation: there is no mention of klipper-v0.12.0-10-gea2f6bc0.bin in the download section.
